src/sass/modules/_m-form-elements.scss

```diff
@@ -75,9 +75,9 @@ button.form-button-disabled {
   // See 18F/web-design-standards#1490
   // This fixes things in the meantime.
   [type="checkbox"] {
-    height: auto !important;
+    height: 1.8rem !important;
```

Comment:
@raquelromano-gov do you have vets api set up locally or are you trying to take a look at the hca app without the backend?